admiral/pkg/clusters/handler.go

```diff
@@ -322,42 +322,23 @@ func handleDestinationRuleEvent(obj *v1alpha3.DestinationRule, dh *DestinationRu
 			}
 
 			if event == common.Delete {
+				deleteDestinationRulePostStateCheck(rc,syncNamespace,obj.Name, clusterId,r.AdmiralState)
```

Comment:
See if we can have the AdmiralState exposed on common package so that we dont have to pass it on between the methods?